app/src/lib/menu-update.ts

```diff
@@ -155,8 +155,15 @@ function getMenuState(state: IAppState): Map<MenuIDs, IMenuItemState> {
     menuStateBuilder.setEnabled('pull', hasPublishedBranch && !networkActionInProgress)
     menuStateBuilder.setEnabled('create-branch', !tipStateIsUnknown)
   } else {
-    for (const id of repositoryScopedIDs) {
-      menuStateBuilder.disable(id)
+    if (selectedState && selectedState.type === SelectionType.MissingRepository) {
+      menuStateBuilder.enable('view-repository-on-github')
+      menuStateBuilder.enable('remove-repository')
+    } else {
```

Comment:
oddly enough, no because first time this code gets hit selectedState is null so they all get disabled and second time this code gets hit only `'view-repository-on-github'` and `'remove-repository'` get enabled